### ISSUE TITLE
PAC3 Certain Models Dont Work.

### ISSUE THREAD

**Issue by RaccoonTheDespoiler** [CapsAdmin/pac3#1300]
Recently Ive Ported New Models, and Tried Old Pacs from a few weeks ago before the update, Now they do not work. No clue as to why.
![broken](https://github.com/CapsAdmin/pac3/assets/140041611/44f48582-e105-43bc-b842-30c2f89cfe2e)
![20230628020016_1](https://github.com/CapsAdmin/pac3/assets/140041611/9afdd6da-c8ad-4a2b-ada3-c5967a5809b4)

**Comment by RaccoonTheDespoiler** [comment]
The Top Image was recent, the bottom is from a month or so ago.

**Comment by OddInc** [comment]
Can confirm also having this problem with PACs dropboxed / hosted outside of gmod. Tried both dropbox and google drive. Only happening with some models. Unknown as to why.

**Comment by OddInc** [comment]
Example of working model: https://www.dropbox.com/s/lgdfdnm953zm7jm/darthrevan.zip?dl=1

Example of non-working model: https://www.dropbox.com/s/t7zvze2z856gyy3/cowboyhat.zip?dl=1

Note: on the non-working model you should be able to see the hat (not the rest of the model), but it is missing textures. All appropriate textures (for the hat only) are included.

**Comment by bjoern90** [comment]
> Example of working model: https://www.dropbox.com/s/lgdfdnm953zm7jm/darthrevan.zip?dl=1
> 
> Example of non-working model: https://www.dropbox.com/s/t7zvze2z856gyy3/cowboyhat.zip?dl=1
> 
> Note: on the non-working model you should be able to see the hat (not the rest of the model), but it is missing textures. All appropriate textures (for the hat only) are included. Non-working model worked at time of dropbox upload.

the model wont work if there's missing materials as the console said so too https://i.imgur.com/iqTOkwn.png

**Comment by OddInc** [comment]
> > Example of working model: https://www.dropbox.com/s/lgdfdnm953zm7jm/darthrevan.zip?dl=1
> > Example of non-working model: https://www.dropbox.com/s/t7zvze2z856gyy3/cowboyhat.zip?dl=1
> > Note: on the non-working model you should be able to see the hat (not the rest of the model), but it is missing textures. All appropriate textures (for the hat only) are included. Non-working model worked at time of dropbox upload.
> 
> the model wont work if there's missing materials as the console said so too https://i.imgur.com/iqTOkwn.png

Yes, but the hat textures ARE included and the hat textures loaded fine in the past. It is a common practice to delete textures you are not using to save size on a dropbox model.

**Comment by Yagira** [comment]
Seems like it's a side effect of another fix, anyway, should work fine in https://github.com/CapsAdmin/pac3/commit/73e4a6f55dcac4cd7075188870eb1c5db9693e12. 

You can try it in singleplayer by downloading the latest PAC version here: https://github.com/CapsAdmin/pac3/archive/refs/heads/develop.zip


> It is a common practice to delete textures you are not using to save size on a dropbox model.

True but you can keep the material files (.vmt), they take almost no space and solve issues like this one. If you want to properly get rid of them then edit the model.
